import-service/src/libs/csv.ts

```diff
@@ -0,0 +1,30 @@
+import csv from "csv-parser";
+import ReadableStream = NodeJS.ReadableStream;
+
+export const fromStream = async (stream: ReadableStream, options: { logContent: boolean; }) => {
+    const parsedCSVRows = [];
+    new Promise((resolve, reject) => {
+        stream.pipe(csv())
+            .on('data', (row) => {
+                if(options.logContent) console.log(row);
```

Comment:
> console.log(row)
Won't it be somthing like [object, Object] if we log it without JSON.stringify?

Reply:
<img width="1034" alt="image" src="https://user-images.githubusercontent.com/37940444/129921511-3a95c70e-9312-4df9-be1c-f3df056e2940.png">

Reply:
it's one level object, so it's fine